### ISSUE TITLE
Add item_data.inc __item_stats_data array to be readily accessible as json data

### ISSUE THREAD

**Issue by Niphyr** [MechanicalPriest/SimcProfileParser#15]
See #10 for information on previous implementations.

Another file needs to be added from this file: https://raw.githubusercontent.com/simulationcraft/simc/shadowlands/engine/dbc/generated/item_data.inc

It's already known in the cache service as `SimcFileType.ItemDataInc`

Structure is an array of `dbc_item_data_t::stats_t` named `__item_stats_data`:

```c++
static const dbc_item_data_t::stats_t __item_stats_data[47463] = {
```
`dbc_item_data_t::stats_t` structure:
```c++
struct stats_t {
    int16_t type_e; // item_mod_type
    int16_t alloc;
    float   socket_mul;
  };
```
This matches the existing class SimcItemMod.

**Comment by Niphyr** [comment]
Added in https://github.com/MechanicalPriest/SimcProfileParser/commit/c135bed01ac81e404575a2f3cde9560ed5e941e0

**Comment by Niphyr** [comment]
Closed with #22
